### DISCUSSION TITLE
Help!!!! I can not save the files

### DISCUSSION THREAD

**Discussion by CxLawrence** [sec-edgar/sec-edgar#314] [Q&A]
![image](https://github.com/user-attachments/assets/7c06eb25-a388-457a-86e4-1724cb1c3902)

I follow the guide to code for capturing the 10-k, but I can not save these files. Could you plz help me and tell me what should I do?

**Comment by jackmoody11** [comment]
Duplicate of #310. Can you try pip installing from the master branch on GitHub?
